ci/axis/rapidsai-base-runtime.yaml

```diff
@@ -3,15 +3,14 @@ BUILD_IMAGE:
   - rapidsai/rapidsai-nightly
 
 FROM_IMAGE:
-  - rapidsai/rapidsai-core-nightly
+  - rapidsai/rapidsai-core
```

Comment:
We shouldn't need to remove the nightly channels, just exclude.

Reply:
> We shouldn't need to remove the nightly channels, just exclude.

This PR matches how we did it in the `0.17` release (see [main](https://github.com/rapidsai/docker/blob/main/ci/axis/rapidsai-base-runtime.yaml#L5-L6)). Also, there should only ever be one `FROM_IMAGE` when we build these images, so it seems counterintuitive to add a second `FROM_IMAGE` and then exclude the first.

Reply:
You may be thinking of the `BUILD_IMAGE` axis value.

Reply:
Ya I got them mixed up.

Reply:
Ya I got them mixed up.